apps/scan/frontend/src/components/absolute.tsx

```diff
@@ -10,9 +10,6 @@ interface Props {
 
 export const Absolute = styled.div<Props>`
   position: absolute;
-  top: ${({ top }) => top && 0};
-  right: ${({ right }) => right && 0};
-  bottom: ${({ bottom }) => bottom && 0};
-  left: ${({ left }) => left && 0};
+  inset: ${({ top }) => top && 0} ${({ right }) => right && 0} ${({ bottom }) => bottom && 0} ${({ left }) => left && 0};
```

Comment:
This is the one time that autofix with the new rules did something dangerous. Since it was only once across the repo I'm not too worried about it, and not super inclined to turn it off, but if you think we should I'm fine with that. I fix this in a subsequent commit.